source/Score.js

```diff
@@ -0,0 +1,60 @@
+import Reference from "./Reference.js"
+
+
+export default class Score {
+    constructor(playerNumber) {
+        this.count = 0
+        this.playerNumber = (playerNumber === undefined) ? game.playerCount : playerNumber
+        this.domElement = document.createElement("div")
+        this.domElement.className += " player-score"
+        this.domElement.style.position = "absolute"
+        this.domElement.style.textAlign = "center"
+        var whiteText = document.createElement("div")
+        var colorText = document.createElement("div")
+        whiteText.className += " foreground-score"
+        colorText.id = "player-" + this.playerNumber + "-score"
+        colorText.className += " shadow-score"
+        this.domElement.appendChild(whiteText)
+        this.domElement.appendChild(colorText)
+        document.body.appendChild(this.domElement)
+        this.gameCanvas = document.getElementById("gameCanvas")
+        this.update()
```

Comment:
Should we bring in a DOM library for initialization and manipulation stuff like this? It would make the code more maintainable.